File: Easy Problems/Leetcode 3731.cpp
class Solution {
public:
    vector<int> findMissingElements(vector<int>& nums) {
        sort(nums.begin(),nums.end());

        vector<int> ans;

        int temp = nums[0];
        int i = 0;
        while(i < nums.size()){
            if(nums[i] == temp){
                temp++;
                i++;
            }
            else{
                ans.push_back(temp++);
            }
        }
        return ans;
    }
};

File: Easy Problems/Leetcode 3731.py
class Solution:
    def findMissingElements(self, nums: List[int]) -> List[int]:
        mn = min(nums)
        mx = max(nums)
        s = set(nums)
        res = []
        for i in range(mn,mx):
            if i not in s:
                res.append(i)
        return res